src/logger.js

```diff
@@ -1,5 +1,15 @@
-import request from "request";
+import request from "browser-request";
 
-export default function (message) {
-  request.post(`${window.location}accesslint`, message)
+export default function (message, url, raiseError) {
+  let callback = function(err) {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).